### PR TITLE
Create lt_LT.js

### PR DESCRIPTION
**Pull request by noen**
Translation for Lithuanian language

### CODE REVIEW

**Comment by amsul** [comment]
Sweet, thanks for this :+1:

File: lib/translations/lt_LT.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    labelYearSelect: 'Pasirinkite metus',
+    monthsFull: ['Sausis', 'Vasaris', 'Kovas', 'Balandis', 'Gegužė', 'Birželis', 'Liepa', 'Rugpjūtis', 'Rugsėjis', 'Spalis', 'Lapkritis', 'Gruodis'],
+    monthsShort: ['Sau', 'Vas', 'Kov', 'Bal', 'Geg', 'Bir', 'Lie', 'Rgp', 'Rgs', 'Spa', 'Lap', 'Grd'],
+    weekdaysFull: ['Sekmadienis', 'Pirmadienis', 'Antradienis', 'Trečiadienis', 'Ketvirtadienis', 'Penktadienis', 'Šeštadienis'],
+    weekdaysShort: ['Sk', 'Pr', 'An', 'Tr', 'Kt', 'Pn', 'Št'],
+    today: 'Šiandien',
+    clear: 'Išvalyti',
+    close: 'Uždaryti',
+    firstDay: 1,
+    //format: 'yyyy !m. mmmm d !d.', // need to have diffrent case of full months name
```
Comment (amsul) [inline lib/translations/lt_LT.js]: Can u explain what you mean by this?

Comment (noen) [inline lib/translations/lt_LT.js]: When it's a simple month name is in Lithuanian: Sausis, Vasaris... but if you output full length date it must be like: 2015 m. sausio 1 d. (genitive). More about this http://en.wikipedia.org/wiki/Lithuanian_declension

Comment (amsul) [inline lib/translations/lt_LT.js]: I see...cool, let’s stick with the one you used then (same as here: http://en.wikipedia.org/wiki/Date_format_by_country)

:+1:

**Pull request merged**